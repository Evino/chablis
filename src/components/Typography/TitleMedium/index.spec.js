import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';
import TitleMedium from '.';

const text = 'This is a SubTitle';
const element = <TitleMedium>{text}</TitleMedium>;
const component = mount(element);

describe('TitleLarge', () => {
  it('should render a subtitle', () => {
    const tree = renderer
      .create(element)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('should display passed text', () => {
    expect(component.prop('children')).toBe(text);
  });
});
