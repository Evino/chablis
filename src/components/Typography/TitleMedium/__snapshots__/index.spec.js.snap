// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleLarge should render a subtitle 1`] = `
<h4
  className="TitleMedium__SubTitleComponent-sc-105k1zd-0 hLEnuP"
>
  This is a SubTitle
</h4>
`;
